.github/workflows/pages.yml

```diff
@@ -18,17 +18,19 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Setup Pages
-        uses: actions/configure-pages@v2
-      - name: Build with Jekyll
-        uses: actions/jekyll-build-pages@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: yarn
+      - run: yarn install --immutable
+      - run: yarn docs
+      - uses: actions/configure-pages@v2
+      - uses: actions/jekyll-build-pages@v1
         with:
           source: ./
           destination: ./_site
-      - name: Upload artifact
-        uses: actions/upload-pages-artifact@v1
+      - uses: actions/upload-pages-artifact@v1
```

Comment:
@JoseLion I noticed you removed the `name` property from these steps, any reason for doing that?

Reply:
Only for the sake of keeping consistency with the other workflow `.yml` files. I do prefer having names on the step, but we can do that for all workflows on a separate PR to maintain consistency 🙂

Reply:
Sounds good!